api/app/fixtures/init_data.yaml

```diff
@@ -0,0 +1,263 @@
+- model: sites.site
+  pk: 1
+  fields:
+    domain: example.com
+    name: example.com
+- model: organisations.organisation
+  pk: 1
+  fields:
+    name: Test Org
+    has_requested_features: false
+    webhook_notification_email: null
+    created_date: 2021-08-02 12:23:25.410055+00:00
+    alerted_over_plan_limit: false
+    stop_serving_flags: false
+    persist_trait_data: true
+    block_access_to_admin: false
+    feature_analytics: false
+- model: organisations.userorganisation
+  pk: 1
+  fields:
+    user: 1
+    organisation: 1
+    date_joined: 2021-08-02 12:23:25.419702+00:00
+    role: ADMIN
+- model: invites.invitelink
+  pk: 1
+  fields:
+    hash: bRRc9MSLgoifYFyqBcSUsK
+    date_created: 2021-08-02 12:23:26.850004+00:00
+    organisation: 1
+    role: ADMIN
+    expires_at: null
+- model: invites.invitelink
+  pk: 2
+  fields:
+    hash: UVySd6hw2sciGyj6c6mStb
+    date_created: 2021-08-02 12:23:26.853142+00:00
+    organisation: 1
+    role: USER
+    expires_at: null
+- model: projects.project
+  pk: 1
+  fields:
+    name: Test Project
+    created_date: 2021-08-02 12:23:33.981797+00:00
+    organisation: 1
+    hide_disabled_flags: false
+- model: projects.userprojectpermission
```

Comment:
This will break if there is no user. I don't think we need to worry about userprojectpermission at all.

Reply:
> To load the data on fresh database(after creating a superuser) run the below command.
I thought the user will be there?

Reply:
The endpoints to get flags will work. All that the client would need to do to get the rest of the application working is to create a user and assign them as an organisation admin. They don't need to create these permission entities.